src/battle_ai_util.c

```diff
@@ -473,6 +473,14 @@ bool32 IsBattlerAIControlled(u32 battlerId)
     }
 }
 
+bool32 IsBattlerAIControlled(u32 battlerId)
```

Comment:
Wouldn't this cause issues with how AI detects if a Pokémon is not controlled by a player?

Reply:
It's not used in that many places. And the relevant places have been replaced with BattlerHasAi

Reply:
It's better now, but the name doesn't sit well for me.
If I call this function with a player-controlled battlerId and the omniscient flag is on, the function name should conceptually still return FALSE do to that battler not being player-controlled, but it's not doing that currently.